### ISSUE TITLE
Accessibility: Link text used for multiple different destinations - Home Links on Exhibit

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#417]
**Description**
The "Home" link in the Exhibit navigation and the "Home" link in the breadcrumbs are going to two different locations. An [example report for this issue can be found here](https://my2.siteimprove.com/Inspector/602695/Accessibility/Page?pageId=51679784912&impmd=3CC33A985F387DC37EA87D3D5A2306C4#/Criterion/2.4.4/Check/39). 

The "Home" navigation link goes to https://exhibits.library.cornell.edu/flights-of-fancy
The "Home" breadcrumb link goes to https://exhibits.library.cornell.edu/flights-of-fancy?q=

**Solutions**
Change the breadcrumb "Home" link to match the exhibit home page link.

**Comment by elrayle** [comment]
NOTE: This is an item page. 

Example:   https://exhibits.library.cornell.edu/flights-of-fancy/catalog/76-4246

**Comment by elrayle** [comment]
Addressed PR https://github.com/projectblacklight/spotlight/pull/2656.

Closing of this issue is pending:
* merge in Spotlight master
* backport to Spotlight release-2.x
* new release of Spotlight 2.x series
* update of our Spotlight to the new 2.x release

**Comment by elrayle** [comment]
There was a comment in the original code that the `q: ''` is required, but it isn't clear what happens if it isn't included.  The comment was written in 2014, so it could be that it is not an issue in the latest version of Rails.  But I need to figure out which version was the problem so a pin can be put in place to make sure someone can't accidentally hit a problem with the change.

**Comment by chrisrlc** [comment]
PR already merged into spotlight and spotlight was upgraded. No longer an issue in Site Improve.
